control-plane-operator/hostedclusterconfigoperator/controllers/hcpstatus/hcpstatus.go

```diff
@@ -94,6 +91,16 @@ func (h *hcpStatusReconciler) reconcile(ctx context.Context, hcp *hyperv1.Hosted
 	err := h.hostedClusterClient.Get(ctx, crclient.ObjectKey{Name: "version"}, &clusterVersion)
 	// We check err in loop below to build conditions with ConditionUnknown status for all types.
 
+	if err != nil {
+		hcp.Status.Version = &hyperv1.ClusterVersionStatus{
```

Comment:
you're deprecating .Version in favour of .VersionStatus but still using .Version here, what's the plan?

Reply:
This was just an incomplete transition to the new names, now that Cesar [recommended `VersionStatus`][1].  I've caught this section up with the `VersionStatus` pattern in d2909321 -> 7de6adde.

[1]: https://github.com/openshift/hypershift/pull/1954#discussion_r1065247903